client/src/locales/en-US.json

```
@@ -1087,6 +1087,7 @@
                 "summary": "The device {label} has been added to your profile.",
                 "getDeviceInfo": "Waiting for you to fill the information on the guest device.",
                 "waitForGuest": "Parsec must be open on both devices during the onboarding process.",
+                "waitForGuestVersionInfo": "If you're having trouble during the device creation process, make sure you're using the latest version of Parsec on both devices.",
```

Why not check it every time?
```suggestion
                "waitForGuestVersionInfo": "Before you start, make sure you are using the latest version of Parsec on both devices.",
```